src/rngToolsUi/gen3/rstid/rstid.tsx

```diff
@@ -0,0 +1,117 @@
+import { Gen3TidSidResult, rngTools } from "~/rngTools";
+import {
+  Field,
+  FormikIdFilter,
+  FormikNumberInput,
+  ResultColumn,
+  RngToolForm,
+  RngToolSubmit,
+  Button,
+  Icon,
+} from "~/components";
+import React from "react";
+import { denormalizeIdFilter, IdFilterSchema } from "~/types/id";
+import { z } from "zod";
+
+type Result = Gen3TidSidResult & { time: string };
+
+const getColumns = (): ResultColumn<Result>[] => [
```

Comment:
This doesn't need to be a function
```suggestion
const columns: ResultColumn<Result>[] = [
```